.github/workflows/debug-artifacts.yml

```
@@ -55,6 +55,9 @@ jobs:
         uses: ./.github/prepare-test
         with:
           version: ${{ matrix.version }}
+      - uses: actions/setup-go@v3
+        with:
+        go-version: ^1.13.1
```

I think this is why these checks aren't running.
```suggestion
          go-version: ^1.13.1
```